documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -218,7 +218,7 @@
               # https://www.robustperception.io/alerting-on-gauges-in-prometheus-2-0 for details.
               (
                 max_over_time(prometheus_remote_storage_shards_desired{%(prometheusSelector)s}[5m])
-              > on(job, instance) group_right
+              > on(job, instance, queue)
```

Comment:
Is the `on` clause necessary at all?

Reply:
Right. There is actually no reason why these two should have different labels. and we want to keep all labels anyway. Will fix in next push.